build_wheel.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+#************************************************#
+#           build_wheel.sh                       #
+#    Build a Python3 wheel for YOLACT            #
+#************************************************#
+
+REPO_NAME="yolact"
+NEEDED_FOLDERS=("data" "external" "layers" "scripts" "utils" "web")
+REQUIREMENTS_FILE="requirements.txt"
+SETUP_FILE="setup.py"
+PYTHON_EXTENSION=".py"
+PACKAGE_TEST_DIR="/tmp/$USER/pip_package_test"
+
+# --------------------------------------------------------- #
+# main ()                                                   #
+# drives the entire build process                           #
+# Returns: 0 on success, non-zero if something goes wrong.  #
+# --------------------------------------------------------- #
+main() {
+  echo "Starting build ..."
+  pre_build_setup &&
+    build &&
+    post_build_test_and_teardown &&
+    echo "Build Successful ... "
+}
+
+# --------------------------------------------------------- #
+# pre_build_setup ()                                        #
+# performs necessary setup for building a YOLACT wheel      #
+# Returns: 0 on success, non-zero if something goes wrong.  #
+# --------------------------------------------------------- #
+pre_build_setup() {
```

Comment:
I'm not sure I disagree with your choice of variable name, but I do wonder, why name these other variables relative to `build`? I can't decide if `pre_build_setup` tells me something more than `setup` does.